### DISCUSSION TITLE
Work in progress any tips?

### DISCUSSION THREAD

**Discussion by vinzify** [mathoudebine/turing-smart-screen-python#121] [Themes]
Hi there! I'm currently studying Python and I'm working on a template that includes all the sensors available. I was wondering if anyone could give me some guidance on how to add additional sensors to the template. Additionally, I'm experiencing some issues with the Upload/download stats not working as expected, and I'm wondering if anyone might have some ideas on how to troubleshoot this problem. Any suggestions or advice would be greatly appreciated! (I'll be happy to share the template once I finish it.)

![2022-12-29-18 18 16-localhost_5678---Brave](https://user-images.githubusercontent.com/12836992/209987380-7a659099-1e11-4665-a341-d2f9adb90c4d.png)

**Comment by mathoudebine** [comment]
Hi @twinalien 
Very nice work with your theme! Feel free to share it with the community in [this discussion space](https://github.com/mathoudebine/turing-smart-screen-python/discussions/categories/themes) when it's done!
Concerning your network statistics not working, did you update `config.yaml` with your interfaces names?
The Python console output of the program would also be helpful to understand the issue

**Comment by vinzify** [comment]
Thanks for the answer @mathoudebine, as you stated I didn't notice that setting and I was able to setup the right network name. I'm building also a template in Illustrator to help people customize it easily with coordinate references. I'll share it as soon as possible with a little guide.

It would be great to have a quick chat with you to get some direction on how I can work on some integration. I would love to have more things like Spotify or fun rpm. Maybe I can contribute :)

![full](https://user-images.githubusercontent.com/12836992/210183600-fab967cb-685b-444a-b3bf-cda486dd6435.png)
